webpack.config.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+
+module.exports = {
+  optimization: {
+    minimize: true
+  },
+  mode: 'production',
+  entry: './mqtt.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist')
+  }
+}
```

Comment:
Well the test failures don't have anything to do with the build.  So if that can be fixed you're good to go.